src/main/java/com/agricraft/agricore/plant/AgriProduct.java

```diff
@@ -40,7 +40,7 @@ public boolean validate() {
         if (!super.validate()) {
             AgriCore.getCoreLogger().info("Invalid Product: Invalid Item: {0}!", item);
             return false;
-        } else if (min < 1) {
+        } else if (min < 0) {
```

Comment:
The log message here will need to be updated.

Reply:
Well spotted